.github/scripts/install-arduino-core-esp32.sh

```diff
@@ -15,13 +15,14 @@ if [ ! -d "$ARDUINO_ESP32_PATH" ]; then
 		pip install requests > /dev/null
 	fi
 
-	if [ "$GITHUB_REPOSITORY" == "espressif/arduino-esp32" ];  then
-		echo "Linking Core..."
-		ln -s $GITHUB_WORKSPACE esp32
-	else
-		echo "Cloning Core Repository..."
-		git clone https://github.com/espressif/arduino-esp32.git esp32 > /dev/null 2>&1
-	fi
+# I don't get the code below, you'd want to test the source code in the current repository (forked or not), not in the espressif repo right?
+#if [ "$GITHUB_REPOSITORY" == "espressif/arduino-esp32" ];  then
+	echo "Linking Core..."
+	ln -s $GITHUB_WORKSPACE "$ARDUINO_ESP32_PATH"
+#else
+#	echo "Cloning Core Repository ..."
+#	git clone --recursive https://github.com/espressif/arduino-esp32.git "$PLATFORMIO_ESP32_PATH" > /dev/null 2>&1
+#fi
```

Comment:
please do not edit those scripts. They are used by CI

Reply:
The scripts as they where did not pull my repo and therefore tested the espressif/arduino-esp32 git repository, excluding my code changes. I think this is undesired behaviour for a CI pipeline as a CI pipeline is used to test the current repository and should be setup to test the repository that the configuration file is stored in.

Reply:
I'll remove the commented lines, but I won't revert back to the original script as that doesn't make sense at all.